test/AudioPlayr/addEventImmediate.ts

```diff
@@ -0,0 +1,36 @@
+/// <reference path="../../node_modules/@types/chai/index.d.ts" />
+/// <reference path="../../node_modules/@types/mocha/index.d.ts" />
+/// <reference path="../../lib/AudioPlayr.d.ts" />
+/// <reference path="../utils/MochaLoader.ts" />
+/// <reference path="../utils/mocks.ts" />
+
+mochaLoader.addTest("calls the function immediately if no sound is found", (): void => {
```

Comment:
Before I forget: a slightly better name would be if the sound doesn't exist yet.